examples/clients/python/setup.py

```diff
@@ -0,0 +1,13 @@
+from distutils.core import setup
+
+setup(
+    name="python-seafowl-client",
+    version="0.0.1",
+    description="Python client for the Seafowl analytical SQL database.",
+    license="Apache 2.0",
+    author="Splitgraph, Inc",
+    author_email="hello@splitgraph.com",
+    url="https://github.com/matiascodesal/git-for-pip-example",
+    packages=["seafowl"],
+    install_requires=["requests"],
```

Comment:
Doesn't this need `sqlalchemy` (maybe we can avoid pulling it in) and `pandas`?

Reply:
Not if you're just using `query()`.

Reply:
Oh, I see, this won't crash if the user only imports `seafowl.seafowl`. Maybe add it as an extra?